samples/Localization.StarterWeb/Localization.StarterWeb.csproj

```diff
@@ -42,7 +34,6 @@
 
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.2.0-*" />
-    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.1-*" />
```

Comment:
My point was `restore` of the web site project works with my changes in place. But, I'll revert this removal.